certbot-apache/local-oldest-requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Remember to update setup.py to match the package versions below.
 acme[dev]==0.29.0
-certbot[dev]==0.39.0
+-e certbot[dev]
```

Comment:
This shouldn't be needed if we change back the name of the lockfile, but if you want to keep it for some reason, I think this should be `-e .[dev]` and we should update `certbot-apache/setup.py` as well.

Reply:
interesting -- this is technically true, but apache does refer in comments to `_internal`. shouldn't matter either way, really.

Reply:
That's a good point. I'd vote to bump the minimum version then so that's always a valid path to an object but I don't care too much.

(The object being referenced is private which I think is another problem and something I missed in my initial design, but I personally think we can punt this issue down the road.)

Reply:
I already reverted; if we decide it's actually a problem that the object referenced is private we'll end up bumping later anyway.